kartothek/io_components/metapartition.py

```diff
@@ -646,6 +646,15 @@ def load_dataframes(
         """
         if columns is None:
             columns = {}
+        elif set(columns).difference(self.files.keys()):
+            raise (
+                ValueError(
+                    "You are trying to read columns from table that does not exist. Valid tables are {}".format(
+                        list(self.files.keys())
```

Comment:
I like what you're doing here. 
Nitpick regarding error message:  show the tables that are invalid (`set(columns).difference(self.files.keys())`).